eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -117,12 +119,13 @@ public void setActiveEditorPart(IEditorPart partRef) {
         if (!(partRef.getEditorInput() instanceof IFileEditorInput)) {
             return;
         }
-
+        if (partRef == activeEditorPart) {
+            return;
+        }
```

Comment:
Are you sure this won't cause any side effects, for example after building the project? This early return means the report list won't be refreshed when the active editor is the same as before, and that's probably not what's expected from this method. Maybe this condition would be better at the inner if for the BugPathListView?